File: qt/Meteorienteer/taivaanvahtiform.cpp
#include "taivaanvahtiform.h"

TaivaanvahtiForm::TaivaanvahtiForm(QObject *parent) :
    QObject(parent)
{
}

void TaivaanvahtiForm::setFields(QVector<TaivaanvahtiField *> fields)
{
    m_fields = fields;
}

QVector<TaivaanvahtiField *> TaivaanvahtiForm::fields()
{
    return m_fields;
}

File: MeteorienteerU/Assets/Scripts/TaivaanvahtiForm.cs
using UnityEngine;
using System.Collections.Generic;

public class TaivaanvahtiForm
{
	public List<TaivaanvahtiField> Fields {get; private set;}
	
	public void SetFields(List<TaivaanvahtiField> fields)
	{
		Fields = fields;
	}
}
